crates/bevy_render/src/lib.rs

```diff
@@ -274,6 +279,11 @@ impl Plugin for RenderPlugin {
                         .unwrap();
                     cleanup.run(&mut render_app.world);
 
+                    render_app
+                        .world
+                        .resource_mut::<NumberOfRenderingEntitiesToReserve>()
+                        .0 = render_app.world.entities().meta.len() as u32;
```

Comment:
```suggestion
                    let mut entities_to_reserve = render_app
                        .world
                        .resource_mut::<NumberOfRenderingEntitiesToReserve>();
                    entities_to_reserve.0 = render_app.world.entities().meta.len() as u32;
```
The PR looks good. But I find assignments at the end of field access chains a bit difficult to read. It could be improved this way.